### PR TITLE
#295 Timer depends on local time

### PR DESCRIPTION
**Pull request by SponsoredByPuma**
Timer is no longer based on the Date time (System time).
It decrements the counter every second, instead of calculating based of the system time every 100 ms.

### CODE REVIEW

**Review by KzuDemEvin — COMMENTED** [review]
Is the timestamp still required?

**Comment by SponsoredByPuma** [comment]
@KzuDemEvin We still need the timeStamp or else the timer would not go back up to its origin value, when you press the restart button as the host.

File: backend/src/main/java/io/diveni/backend/controller/WebsocketController.java [hunk 1]
```diff
@@ -232,4 +236,16 @@ public synchronized void adminSelectedUserStory(
...
+  @GetMapping("/get-timer-value")
+  public synchronized ResponseEntity<Long> getTimeValue(String memberID) throws ParseException {
+    LOGGER.debug("--> get-timer-value");
+    Session session =
+        ControllerUtils.getSessionByMemberIDOrThrowResponse(databaseService, memberID);
+    Date startDate = Utils.getDateFromString(session.getTimerTimestamp());
+    Date currentDate = new Date();
+    long timeDifference = currentDate.getTime() - startDate.getTime();
+    LOGGER.debug("<-- get-timer-value");
+    return new ResponseEntity<>(timeDifference, HttpStatus.CREATED);
```
Comment (stritti) [inline backend/src/main/java/io/diveni/backend/controller/WebsocketController.java:249]: Why status `CREATED`? Should it not be `OK`?

**Review by stritti — APPROVED** [review]

File: backend/src/main/java/io/diveni/backend/controller/WebsocketController.java (hunk 1 above)
Comment (SponsoredByPuma) [reply backend/src/main/java/io/diveni/backend/controller/WebsocketController.java]: Yeah you're right. Thanks I didnt notice it.

**Pull request merged**